starters/basic-starter/lib/drupal.ts

```diff
@@ -0,0 +1,8 @@
+import { DrupalClient } from 'next-drupal';
+
+export const drupal = new DrupalClient(
+  process.env.NEXT_PUBLIC_DRUPAL_BASE_URL,
+  {
+    previewSecret: process.env.DRUPAL_PREVIEW_SECRET,
```

Comment:
I'm getting following error when using the preview mode:

```
$ yarn start
yarn run v1.22.17
$ next start
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from /Users/lauri.eskola/Projects/next-acquia-cms/starters/basic-starter/.env.local
Error: auth is not configured. See https://next-drupal.org/docs/client/auth
    at /Users/lauri.eskola/Projects/next-acquia-cms/node_modules/next-drupal/dist/index.js:1177:23
    at /Users/lauri.eskola/Projects/next-acquia-cms/node_modules/next-drupal/dist/index.js:1285:12
    at DrupalClient.<anonymous> (/Users/lauri.eskola/Projects/next-acquia-cms/node_modules/next-drupal/dist/index.js:1287:8)
    at DrupalClient.fetch (/Users/lauri.eskola/Projects/next-acquia-cms/node_modules/next-drupal/dist/index.js:1121:21)
    at DrupalClient.translatePath (/Users/lauri.eskola/Projects/next-acquia-cms/node_modules/next-drupal/dist/index.js:1927:38)
    at DrupalClient.translatePathFromContext (/Users/lauri.eskola/Projects/next-acquia-cms/node_modules/next-drupal/dist/index.js:1950:38)
    at getStaticProps (/Users/lauri.eskola/Projects/next-acquia-cms/starters/basic-starter/.next/server/pages/[...slug].js:136:107)
    at Object.renderToHTML (/Users/lauri.eskola/Projects/next-acquia-cms/node_modules/next/dist/server/render.js:492:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async doRender (/Users/lauri.eskola/Projects/next-acquia-cms/node_modules/next/dist/server/base-server.js:901:38)
```

I think we still need to configurate the authentication here using instructions from https://next-drupal.org/docs/authentication